site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -171,41 +179,54 @@ They can include JavaScript files, CSS files, or both. All auto-run content scri
   </tbody>
 </table>
 
-{% if false %}
 ### Inject with dynamic declarations {: #dynamic-declarative }
 
-{% Aside 'caution' %}
-
-This feature is not yet fully supported. It is currently in dev and is also available in Chrome
-Canary.
-
-{% endAside %}
-
-You should use dynamic declarations in the following cases:
+Dynamic content scripts are useful for situations when the match patterns for a content scripts are
+not well known or when content scripts should not always be injected on known hosts.
 
-- When the host is not well known
-- The script may need to be added/removed from a known host
+Introduced in Chrome 96, dynamic declarations are similar to [static declarations][14], but the
+content script object is registered with Chrome using methods on the Scripting API rather than in
+[manifest.json][doc-manifest]. In addition to [registering][api-register-cs] content scripts, the
+Scripting API also allows extension developers to [get a list of][api-get-registered-cs],
+[update][api-update-cs], and [remove][api-remove-cs] content script registrations.
 
-**TODO**
+Like static declarations, dynamic declarations can include JavaScript files, CSS files, or both.
+Unlike static declarations, dynamic content scripts cannot be [injected into related
+frames][header-related-frames].
```

Comment:
We intend to change this very soon.  I'd omit it (astute developers will see that the property is not on the registration object)

Reply:
Ack, dropping this sentence. I had planned to update this section once support landed, but I can see how this could lead to unnecessary confusion.